Shared Framework Code/API/NightscoutAppDataManager.swift

```diff
@@ -11,36 +11,49 @@ import Foundation
 let updateInterval: NSTimeInterval = Constants.NotableTime.StandardRefreshTime
 public let queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0)
 
+
+/**
+ Fetch the metadata for a given site.
+ The handler will always return on the main thread
+ 
+ - parameter site: Destination site for which we are querying
+ - parameter handler: closure with the resulting site or an error as parameters
+
+ 
+ - returns: none
+ */
 public func quickFetch(site: Site, handler: (returnedSite: Site, error: NightscoutAPIError) -> Void) {
     dispatch_async(queue) {
         print(">>> Entering \(#function) <<<")
-        print("STARTING:    Load all available site data for: \(site.url)")
+        print("STARTING quickFetch:    Load all available site data for: \(site.url)")
```

Comment:
There is a little bit of formatting that I added so that I could see a bit more how things were playing out.